### PR TITLE
Fix export

### PR DESCRIPTION
**Pull request by remcohaszing**
### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 
*   [x] If applicable, I’ve added docs and tests

### Description of changes

`configureUnifiedLanguageServer` was exported, `createUnifiedLanguageServer` is supposed to be exported instead.

### CODE REVIEW

**Comment by wooorm** [comment]
Maybe add a test for it, too? https://github.com/unifiedjs/unified-language-server/blob/fc82fbe4df16c25480861e16951b9840bfbe2b30/test/index.js#L15

**Comment by remcohaszing** [comment]
I’m not sure what such a test looks like. This is more configuration than code really.

**Comment by wooorm** [comment]
Something like https://github.com/remarkjs/remark-rehype/blob/3a9bd089953dc12b82b8403d2bc731ea957c5895/test.js#L8-L14.

And this is using an export map, so you can use the package name to get the public api? `import whateverYouWant from 'unified-language-server'`?

File: package.json [hunk 1]
```diff
@@ -46,8 +46,8 @@
     "@types/tape": "^4.0.0",
     "c8": "^7.0.0",
     "prettier": "^2.0.0",
-    "remark-cli": "^10.0.0",
-    "remark-preset-wooorm": "^9.0.0",
+    "remark-cli": "^10.0.1",
+    "remark-preset-wooorm": "^9.1.0",
```
Comment (wooorm) [inline package.json:50]:
```suggestion
    "remark-cli": "^10.0.0",
    "remark-preset-wooorm": "^9.0.0",
```

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
released!

**Pull request merged**